src/main/webapp/app/courses/results/result.component.ts

```diff
@@ -32,13 +23,13 @@ export class SafeHtmlPipe implements PipeTransform {
 export class ResultComponent implements OnInit, OnChanges, OnDestroy {
 
     @Input() participation: Participation;
-    // @Output() onNewResult;
+    @Input() building: boolean;
```

Comment:
would be more consistent, if we call it isBuilding (another variable in a different file is also called isBuilding)